### PR TITLE
add import section in create product

### PR DESCRIPTION
**Pull request by alexandrelissowski**
new section in the create page to explain the import process.

### CODE REVIEW

File: content/md/supplier-onboarder/create-products.md [hunk 1]
```diff
@@ -29,3 +29,22 @@ The family list will give you access to all the product families you can create
...
 :::
+
+## Speed up with import
+Now, you know how to create a new product manually, you may think there should be a faster way to do it.
+
+The answer is yes!
+
+You can run an import and it's very easy.
+On your grid:
+1. Download the import (see [Download a product import template](dwl-product-import-tpl.html)),
```
Comment (bgouraud) [inline content/md/supplier-onboarder/create-products.md]: "Download the import template" --> I would have added 'template' here.

File: content/md/supplier-onboarder/create-products.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ popular: true
 ---
 
 # Product creation? Piece of cake!
-
+## Step by step clicks
```
Comment (bgouraud) [inline content/md/supplier-onboarder/create-products.md]: Not sure of the expression.
Maybe something like: "Step by step via the UI" or "Manual creation"?

File: content/md/supplier-onboarder/create-products.md [hunk 3]
```diff
@@ -29,3 +29,22 @@ The family list will give you access to all the product families you can create
...
+1. Download the import (see [Download a product import template](dwl-product-import-tpl.html)),
+1. Complete the spreadsheet,
+1. On the product grid, click on the `Import` button,
+1. Drag and drop your file,
+1. Run the import and
+1. Wait for the notification
+
+
+::: warning
+Remember to specify a SKU for all your products to identify them.
```
Comment (bgouraud) [inline content/md/supplier-onboarder/create-products.md]: Not sure this warning is relevant. I'm not sure you can import a product without a SKU.

Comment (alexandrelissowski) [reply content/md/supplier-onboarder/create-products.md]: removed

File: content/md/supplier-onboarder/create-products.md (hunk 2 above)
Comment (alexandrelissowski) [reply content/md/supplier-onboarder/create-products.md]: go for "Manual creation"

**Pull request merged**